scripts/error-codes/dev-expression-with-codes.js

```diff
@@ -35,34 +35,14 @@ module.exports = function(babel) {
     return localState.prodInvariantIdentifier;
   }
 
-  var DEV_EXPRESSION = t.binaryExpression(
-    '!==',
-    t.memberExpression(
-      t.memberExpression(t.identifier('process'), t.identifier('env'), false),
-      t.identifier('NODE_ENV'),
-      false
-    ),
-    t.stringLiteral('production')
-  );
+  var DEV_EXPRESSION = t.identifier('__DEV__');
 
   return {
     pre: function() {
       this.prodInvariantIdentifier = null;
     },
 
     visitor: {
-      Identifier: {
-        enter: function(path) {
-          // Do nothing when testing
-          if (process.env.NODE_ENV === 'test') {
-            return;
-          }
-          // Replace __DEV__ with process.env.NODE_ENV !== 'production'
-          if (path.isIdentifier({name: '__DEV__'})) {
```

Comment:
We can, although this will also require updating a couple of references to this transform in the docs.